packages/rum-core/src/domain/contexts/viewContexts.ts

```diff
@@ -25,6 +26,9 @@ export function startViewContexts(lifeCycle: LifeCycle): ViewContexts {
     viewContextHistory.add(buildViewContext(view), view.startClocks.relative)
   })
 
+  lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, (view) => {
+    viewContextHistory.add(buildViewContext(view), view.startClocks.relative)
+  })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ ‏I'm not sure if it's good thing to add a new history entry on each view update. What about mutating the feature flag object in `trackViews`, so it will be up to date in the context as well?